README.win32.adoc

```diff
@@ -29,7 +29,8 @@ Here is a summary of the main differences between these ports:
 (*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
 version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
 a static linking exception; prior versions of the DLL are distributed under the
-GPL. Thus, these `.exe` files can only be distributed under compatible license.
+GPL (the cygwin version can be obtained with the `uname -r` command). Thus,
+these `.exe` files can only be distributed under a compatible license.
```

Comment:
Sorry for the endless nitpicking, but I (personally) actually prefer the first version of @dra. I find that giving the actual license is much more precise that saying "this version is recommended", which helps people do their own lawyering if they have to, and I think that "may be distributed" is nuanced enough as a formulation to let people know that, if they are picky about this stuff, they may need to check more in depth.